lib/internal/util/inspect.js

```diff
@@ -1061,10 +1079,10 @@ function formatRaw(ctx, value, recurseTimes, typedArray) {
   if (ctx.sorted) {
     const comparator = ctx.sorted === true ? undefined : ctx.sorted;
     if (extrasType === kObjectType) {
-      output = output.sort(comparator);
+      ArrayPrototypeSort(output, comparator);
     } else if (keys.length > 1) {
-      const sorted = output.slice(output.length - keys.length).sort(comparator);
-      output.splice(output.length - keys.length, keys.length, ...sorted);
+      const sorted = ArrayPrototypeSort(ArrayPrototypeSlice(output, output.length - keys.length), comparator);
+      ArrayPrototypeSplice(output, output.length - keys.length, keys.length, ...sorted);
```

Comment:
Can we avoid the spread operator to avoid relying on the globally mutable `Array.prototype[Symbol.iterator]`?
```suggestion
      ArrayPrototypeUnshift(sorted, output, output.length - keys.length, keys.length);
      ReflectApply(ArrayPrototypeSplice, null, sorted);
```